src/chat.rs

```diff
@@ -2034,7 +2034,10 @@ pub async fn create_group_chat(
     });
 
     if protect == ProtectionStatus::Protected {
-        chat_id.set_protection(context, protect).await?;
+        chat_id.inner_set_protection(context, protect).await?;
+        chat_id
+            .add_protection_msg(context, protect, false, 0)
+            .await?;
```

Comment:
set_protection() calls inner_set_protection() and add_protection_msg(), however, we force the message having no creator in the string by giving `0`as contact-id in the last parameter.

the whole thing here will be removed as soon as the ui have a protect/unprotect switch,